spec/musicalbum_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../classes/musicalbum'
+
+
+describe MusicAlbum do
+  before :each do
+    @musictest = MusicAlbum.new(2002-01-01,true,true,true)
```

Comment:
You should pass the date either as a String or as a Date object. You forgot the quotation marks around the date.

Reply:
This has been changed.